### PR TITLE
Update various sections

### PR DESCRIPTION
**Pull request by hestherchang**
made updates to various sections to reflect current processes

Fixes issue(s) # ..n/a. the information was outdated and I updated it

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/18f.gsa.gov/BRANCH_NAME/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/BRANCH_NAME/README.md)

Changes proposed in this pull request:
-
-
-

/cc @relevant-people

### CODE REVIEW

File: _join/index.md [hunk 1]
```diff
@@ -47,29 +47,27 @@ We're building a team that reflects the United States. We don't discriminate bas
...
 
 ## The interview process
 
-The interview process begins with a 30-minute preliminary screening by phone or video call. Plan to talk about your skills and experience, what you're passionate about, and to learn about the work we do here.
+The interview process usually begins with a 30-minute preliminary screening by phone or video call. Plan to talk about your skills and experience and what you're passionate about. We'll also share more about our teams and the work we do here.
 
 If we'd like to chat with you more, we'll invite you to a series of video chats with 18F team members for a couple of longer conversations.
 
-If we agree you're a good fit for the role after interviews, your point of contact on the 18F Talent Team will reach out to you with next steps.
+If a strong match is identified after interviews, your 18F recruiter will reach out to you with next steps.
```
Comment (awfrancisco) [inline _join/index.md]: The first part of this sentence needs a subject. Does 18F identify them as a strong match or do we want to say if both parties identify each other as a strong match?

File: _join/index.md [hunk 2]
```diff
@@ -47,29 +47,27 @@ We're building a team that reflects the United States. We don't discriminate bas
...
+Once the application collection period ends, you can't add to or modify your application.
 
-We begin evaluating applications using a qualification process called [Category Rating](https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/#url=Category-Rating). Following this process, we evaluate each application against specific competencies (you can find these in the job posting) and categorize each as Superior Qualified, Best Qualified, Well Qualified, Qualified, or Not Qualified.
-
-Once we've evaluated all applications within the cohort, we send all the applications and evaluation scores to General Services Administration (GSA) Human Resources, our partner in the hiring process, to apply [Veterans' Preference](http://www.fedshirevets.gov/job/vetpref/index.aspx) for candidates who claim it. GSA HR returns a list of candidates who were categorized as Superior Qualified, and the selection panel (usually the hiring manager) then identifies which candidates to interview.
+A panel of Subject Matter Experts (SMEs) will review each application. The top qualified candidates will be contacted to continue on to the interview process.
```
Comment (awfrancisco) [inline _join/index.md]: The language on the new section if fine, but I'm curious why we're removing a lot of information in favor of this new sentence. If this info is no longer correct, can we give a more detailed version of the current process?

File: _join/index.md (hunk 1 above)
Comment (hestherchang) [inline _join/index.md]: 18F.

18F has to identify the candidate as a a strong match before I can call the candidate and ask them if it's mutual.

File: _join/index.md (hunk 2 above)
Comment (hestherchang) [inline _join/index.md]: The old version has a lot of information that, in my opinion, is overwhelming. Even though I'm run this process and know it very well, this is still a lot for me to process when I read it. The new language still captures what happens in the review process. 

If you think we should keep it as is, that's fine with me. We just need to remove "best qualified, well qualified" to reflect the current process.

Comment (awfrancisco) [inline _join/index.md]: I hear that. We can work to edit it down. Can we keep in the category rating link? And I think this is the only mention of vets preference, so I think we should try to keep some version of that in. I can work up a simpler version if you'd like.

Comment (hestherchang) [inline _join/index.md]: Sure, that sounds great. Thank you.

**Comment by awfrancisco** [comment]
@hestherchang Made some edits. Let me know how that looks.

**Comment by hestherchang** [comment]
@awfrancisco  👍  thank you!

**Pull request merged**